contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -179,6 +180,839 @@ const char *bbf_servicename = "MSSQLSERVER";
 char	   *bbf_language = "us_english";
 #define MD5_HASH_LEN 32
 
+
+static const struct
+{
+	const char *stdname;		/* Windows name of standard timezone */
+	const char *dstname;		/* Windows name of daylight timezone */
+	const char *pgtzname;		/* Name of pgsql timezone to map to */
+}			win32_tzmap[] =
+
+{
+	/*
+	 * This list was built from the contents of the registry at
```

Comment:
let us verify if we are allowed to do this and should we comment the same as well